### ISSUE TITLE
fail2ban role uses incorrect log levels

### ISSUE THREAD

**Issue by runofthemill** [roots/trellis#675]
## Submit a feature request or bug report

- [x] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/guidelines/blob/master/CONTRIBUTING.md)
- [ ] This is a feature request
- [x] This is a bug report
- [x] This request isn't a duplicate of an [existing issue](https://github.com/roots/trellis/issues)
- [x] I've read the [docs](https://roots.io/trellis/docs) and followed them (if applicable)

Replace any `X` with your information.

---

**What is the current behavior?**

Fail2Ban role uses incorrect syntax for log levels:
```
2016-11-01 14:58:53,145 fail2ban.jail           [13268]: INFO    Jail 'ssh' stopped
2016-11-01 14:58:54,144 fail2ban.jail           [13268]: INFO    Jail 'sshd' stopped
2016-11-01 14:58:54,149 fail2ban.transmitter    [13268]: WARNING Command ['set', 'loglevel', '3'] has failed. Received ValueError('Invalid log level',)
2016-11-01 14:58:54,153 fail2ban.server         [13268]: INFO    Changed logging target to /var/log/fail2ban.log for Fail2ban v0.9.3
2016-11-01 14:58:54,155 fail2ban.jail           [13268]: INFO    Creating new jail 'sshd'
```

When trying to reload fail2ban:

```
root@intranet-02:/var/log# fail2ban-client reload
ERROR  NOK: ('Invalid log level',)
```

**What is the expected or desired behavior?**

`fail2ban-client reload` should not produce an error

---

## Bug report

**Where did the bug happen? Development or remote servers?**

Remote

**Other relevant information:**

Per http://www.fail2ban.org/wiki/index.php/Commands:

```
LOGGING
Command		Description
set loglevel <LEVEL>		sets logging level to <LEVEL>. Levels: CRITICAL, ERROR, WARNING, NOTICE, INFO, DEBUG
```

Issue solved by adding `fail2ban_loglevel: INFO` to `group_vars/all/main.yml`, but the role hasn't been updated in +2 years; perhaps consider updating it with a more current role like https://github.com/debops/ansible-fail2ban

**Comment by swalkinshaw** [comment]
Thanks for reporting this. We could just update the log level in https://github.com/roots/trellis/blob/1040828ba54d37b9a4095867067cefe4b5507c6c/roles/fail2ban/defaults/main.yml#L2.

Would like to do that?

**Comment by pelson** [comment]
Sorry for resurrecting. Just wanted to add, that I had a similar issue with fail2ban on homebrew (OSX, v0.8.14).

In my case, the message was caused by the default log location not being writable by the user running fail2ban. My solution (suitable for homebrew) was to change the log location (defined in /usr/local/etc/fail2ban/fail2ban.conf) to have ```logtarget = /usr/local/var/log/fail2ban.log```. I was able to then ```fail2ban-client reload``` without seeing the ```Failed to change log target``` message that I was getting. (upon re-reading, I can see that this *isn't* the same message reported here, but hopefully will be helpful to others who find this thread).
